addons/mail/static/src/core/web/discuss_patch.scss

```diff
@@ -1,3 +1,8 @@
 .o-mail-Discuss-mobileLoading {
     font-size: 70px;
 }
+
+.o-mail-Discuss-headerStartMeeting {
+    width: 300px;
```

Comment:
Hard-coded width probably not a good idea, shouldn't it be linked to other sidebar stuff?

Reply:
(also potential translation overflow)